frameworks/C/libhv/libhv.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
+
+RUN apt-get update -yqq
+RUN apt-get install -y git build-essential
+
+WORKDIR /
+RUN git clone -b tfb https://github.com/ithewei/libhv.git
```

Comment:
You should try to keep any dependencies as locked down as possible, so download `https://github.com/ithewei/libhv/archive/${commit hash}.zip` instead (which also reduces network traffic).